formstack-baton-requests/cloud-formation.yaml

```diff
@@ -154,10 +149,7 @@ Resources:
       Role:
         !GetAtt FormstackBatonSarLambdaRole.Arn
 
-#  There tables are only created in PROD since Formstack doesn't have a CODE environment and we don't want to test in
-#  CODE with real data
```

Comment:
I changed  the table names in the `CODE` stack params so we can let cfn create them